Include/httpClient/config.h

```diff
@@ -99,6 +101,11 @@
 (HC_PLATFORM == HC_PLATFORM_MAC || HC_PLATFORM == HC_PLATFORM_IOS)
 #endif
 
+#if !defined(HC_PLATFORM_IS_EXTERNAL)
```

Comment:
can you add a comment describing what we mean by external here? just to avoid confusion in the future